components/cam/src/physics/rrtmgp/cam_optics.F90

```diff
@@ -63,7 +65,15 @@ subroutine get_cloud_optics_sw( &
 
       integer :: iband, ilev, icol
 
-      ! Initialize
+      ! Initialize outputs
```

Comment:
These initializations to 0s should following the practice of using 0.0_r8. Don't know whenotherwise it would become a problem, though. Similarly for the block starting from line 261.

Reply:
I think that using `0` is not a problem, but using `0.0` could be a problem because `0.0` would be interpreted as single precision. I can change them regardless, but using the integer literals shouldn't cause a problem here.

Reply:
Fixed on the branch, I'll push the commit after testing.

Reply:
Fixed.